### PR TITLE
feat: add the about dex modal

### LINKED ISSUE #313: About page or modal
We need user to be able to see which version and url of 
- DEX, 
- MDW, 
- Node, 
- SDK, 
- compiler

also want to show
- current height

### PR DESCRIPTION
**Pull request by bogdan-manole**
Fixes #313

### CODE REVIEW

File: src/components/AboutModal.vue [hunk 1]
```diff
@@ -0,0 +1,201 @@
+<template>
+  <ModalDefault
+    class="about-dex-modal"
+    :title=" $t('aboutModal.title') "
```
Comment (kenodressel) [inline src/components/AboutModal.vue:4]: why the spaces?

File: src/components/AboutModal.vue [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  }),
+  computed: {
+    ...mapGetters(['activeNetwork']),
+    ...mapState(['sdk']),
+  },
+  async created() {
+    const { sdk } = this;
+    if (sdk) {
+      this.nodeVersion = sdk.selectedNode.version;
+      this.height = await sdk.height();
```
Comment (kenodressel) [inline src/components/AboutModal.vue:130]: should we update this in an interval?

Comment (bogdan-manole) [reply src/components/AboutModal.vue]: done

File: src/components/AboutModal.vue (hunk 1 above)
Comment (bogdan-manole) [reply src/components/AboutModal.vue]: done

**Review by kenodressel — APPROVED** [review]

**Comment by CedrikNikita** [comment]
Adjustments:
- fix warnings
`Should not import the named export 'version' (imported as 'contractsVersion') from default-exporting module (only default export is available soon)`
- fix: show compiler url
- simplify scss and markup
- without await while sdk trully, if user opens fast enough, no depending on sdk information will be presented
- get height before setting an interval so user won't wait till we firstly get it
- increase polling interval, height is not changing that fast (can be even more increased)

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
Commits should be squashed before merge

**Comment by bogdan-manole** [comment]
@CedrikNikita 

> * fix warnings
>   `Should not import the named export 'version' (imported as 'contractsVersion') from default-exporting module (only default export is available soon)`

done

> * fix: show compiler url

what's wrong with it?
image

> * simplify scss and markup

I need more details. This is a modified version of ConfirmLiquidityModal.vue

> * without await while sdk trully, if user opens fast enough, no depending on sdk information will be presented

done (take a look)

> * get height before setting an interval so user won't wait till we firstly get it

done
> * increase polling interval, height is not changing that fast (can be even more increased)

done (set to 15 secs, if you are thinking on sth else please make a suggestion)

**Review by kenodressel — COMMENTED** [review]
some padding on the left column would be nice so the "dex backend" is not directly merging into the url.
image

**Comment by CedrikNikita** [comment]
> some padding on the left column would be nice so the "dex backend" is not directly merging into the url.

done

**Review by kenodressel — APPROVED** [review]

**Review by kenodressel — APPROVED** [review]

**Review by CedrikNikita — APPROVED** [review]

**Pull request merged**